wrapper/fmriprep_docker.py

```diff
@@ -164,8 +164,15 @@ def merge_help(wrapper_help, target_help):
 
     # Make sure we're not clobbering options we don't mean to
     overlap = set(w_flags).intersection(t_flags)
-    expected_overlap = set(['h', 'version', 'w', 'fs-license-file',
-                            'fs-subjects-dir', 'use-plugin'])
+    expected_overlap = set([
+        'anat-derivatives',
+        'fs-license-file',
+        'fs-subjects-dir',
+        'h',
+        'use-plugin',
+        'version',
+        'w',
+    ])
```

Comment:
no need to convert between types
```suggestion
    expected_overlap = {
        'anat-derivatives',
        'fs-license-file',
        'fs-subjects-dir',
        'h',
        'use-plugin',
        'version',
        'w',
    }
```